sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/MeterSharedState.java

```diff
@@ -135,20 +134,20 @@ public final void registerDoubleAsynchronousInstrument(
             .getViewRegistry()
             .findViews(instrument, getInstrumentationLibraryInfo());
     for (View view : views) {
+      MetricStorage currentStorage =
+          AsynchronousMetricStorage.doubleAsynchronousAccumulator(
+              view,
+              instrument,
+              meterProviderSharedState.getResource(),
+              getInstrumentationLibraryInfo(),
+              meterProviderSharedState.getStartEpochNanos(),
+              metricUpdater);
       // TODO - move this in a better location.
-      if (view.getAggregation().config(instrument) == null) {
+      if (AsynchronousMetricStorage.empty().equals(currentStorage)) {
```

Comment:
hmm. :( Kind of an antipattern if you have to ask if your object is the null version.  Could use `==` here, since there should only be one instance, yes? What would be the downisde if we just passed this in to the `register` method?

Reply:
Yeah, I can use `==`.

The downside is you'll get an error around the metric-descriptor duplication.  LIke I said, not super happy with how I handled all this, just don't have a better idea right now.  Will update to ==.

Reply:
we can always iterate. no worries.